todo_app/app.py

```diff
@@ -1,40 +1,96 @@
-from typing import ItemsView
-from flask import Flask
-from .flask_config import Config
-from todo_app.trello_items import init_trello, todolists, doinglists, donelists,new_card,move_doing,move_done
-from flask import Flask, render_template, request, redirect
 from todo_app.view_model import ViewModel
-from todo_app.debugger import writelog
+from .data.item import Item
+from .flask_config import Config
+import os
+from flask import Flask, redirect, render_template, request
+import pymongo
+from bson import ObjectId # For ObjectId to work
+
+#mongodb_host = os.getenv("MONGODB_HOST")
+#mongodb_port = int(os.environ.get('MONGO_PORT', '27017'))
+
+#Configure the connection to the database
+#client = pymongo.MongoClient(mongodb_host) 
+#client = pymongo.MongoClient("mongodb://localhost:27017/")
+
 
 def create_app():
+    dbconnection = os.getenv("MONGODB_CONN")
+    client = pymongo.MongoClient(dbconnection)
+    #Select the database   
+    db = client.todo_app    
+    #Select the collection
+    collection = db.todolist
     app = Flask(__name__)
     app.config.from_object(Config())
     
-    init_trello()
-
+    
+    def todolists():
+        dicttodo=collection.find({"Status" : "To Do"})
+        todos = []
+        for todo_item in dicttodo:
+            todos.append(Item.from_mongo(todo_item))
+        return todos
+        
+    def doinglists():
+            dictdoing=collection.find({"Status" : "In Progress"})
+            doing = []
+            for doing_item in dictdoing:
+                doing.append(Item.from_mongo(doing_item))
+            return doing
+            
+    def donelists():
+            dictdone=collection.find({"Status" : "Completed"})
+            done = []
+            for done_item in dictdone:
+                done.append(Item.from_mongo(done_item))
+            return done
```

Comment:
Minor, but notice the duplication in these functions - what if we made these functions:
```suggestion
    def todolists():
        return get_cards_for_status("To Do")
        
    def doinglists():
        return get_cards_for_status("In Progress")
            
    def donelists():
            return get_cards_for_status("Completed")
    
    def get_cards_for_status(status):
            items = collection.find({"Status" : status})
            filtered_list = []
            for item in items:
                filtered_list.append(Item.from_mongo(item))
            return filtered_list
```